enamdict.go

```diff
@@ -101,11 +101,18 @@ func jmnedictExportDb(inputPath, outputPath, language, title string, stride int,
 		"tag":  jmnedictBuildTagMeta(entities).crush(),
 	}
 
+	index := dbIndex{
+		Title:       title,
+		Revision:    "jmnedict1",
+		Sequenced:   true,
+		Description: "",
```

Comment:
Nit: why initialize fields to a value that is already the default for this and similar conditions? I initially thought that this was to be explicit about struct members, but not all of them are being initialized.